docs/topics/multiplatform/multiplatform-compatibility-guide.md

```diff
@@ -78,6 +78,10 @@ Starting with Gradle 8.7, the Application plugin will no longer work with the Ko
 If you want to use both the Kotlin Multiplatform Gradle plugin and other Gradle plugins for
 Java in your multiplatform project, see [Deprecated compatibility with Kotlin Multiplatform Gradle plugin and Java plugins](multiplatform-compatibility-guide.md#deprecated-compatibility-with-kotlin-multiplatform-gradle-plugin-and-gradle-java-plugins).
 
+If you use the [Java test fixtures](https://docs.gradle.org/current/userguide/java_testing.html#sec:java_test_fixtures) Gradle plugin
+and a Gradle version higher than 8.7, the plugin doesn't work. Instead, upgrade to [Kotlin 2.1.21](releases.md#release-details)
```

Comment:
Maybe I'm taking this out of context, but it reads a bit strange: "instead of using the plugin with Gradle > 8.7, upgrade to Kotlin 2.1.21."

Should we add the older version explicitly? "If you use the plugin, Gradle version higher than 8.7, and Kotlin version below 2.1.21, the plugin doesn't work. Upgrade..."

Reply:
Fair point! Updated the text. WDYT?